app/models.py

```diff
@@ -79,7 +79,7 @@ class Vote(db.Model):
     password = db.Column(db.String())
 
     def __repr__(self):
-        return f'<Vote {self.id}>'
+        return f'<Vote {self.election} {self.candidate} {self.user}>'
```

Comment:
I'm not sure how this works though :)

Reply:
It does. It's basically how it'll appear if your query the Vote table

Reply:
Before it was showing only the id and I didn't like it cos it wasn't self explanatory so I changed it to show the name of the election, the candidate (you're voting for) and the user that's voting